revisions-extended/includes/rest-revisions-controller.php

```diff
@@ -313,6 +318,8 @@ public function filter_rest_prepare_revision( $response, $post, $request ) {
 			$post_type = get_post_type( $parent );
 
 			if ( $parent && $post_type ) {
+				$response->remove_link( 'parent' );
```

Comment:
@StevenDufresne this might be a breaking change for the JS side, I'm not sure.

This removes the first, inaccurate parent link before adding the accurate one, so that there is only one parent link total.

Reply:
Provided a parent link is returned, it should be fine. Does this introduce that chance of no parent being returned?

Reply:
No, there should always be a parent link. There just won't be two anymore. I was mostly concerned that there might be a place somewhere in the JS where it was using an array index of `1` to get the right data...

Reply:
Should be ok:
https://github.com/coreymckrill/revisions-extended/blob/be6e5e3581c627d8e2f2156b9df037c6547c58eb/revisions-extended/src/hooks/parent-post.js#L19